src/compiler/sys.ts

```diff
@@ -620,6 +623,9 @@ namespace ts {
                     if (process.stdout && process.stdout._handle && process.stdout._handle.setBlocking) {
                         process.stdout._handle.setBlocking(true);
                     }
+                },
+                base64decode: input => {
+                    return Buffer.from(input, "base64").toString("utf8");

```

Comment:
`Buffer.from` was [added in NodeJS v5.10.0](https://nodejs.org/dist/latest-v9.x/docs/api/buffer.html#buffer_class_method_buffer_from_string_encoding). We should feature test for `Buffer.from` and either don't add `base64decode` if it isn't present, or fall back to the deprecated `new Buffer(string, encoding)` form that was used prior to NodeJS v6.0.0.